src/ci/citool/tests/test-jobs.yml

```diff
@@ -27,7 +27,7 @@ runners:
     <<: *base-job
 envs:
   env-x86_64-apple-tests: &env-x86_64-apple-tests
-    SCRIPT: ./x.py --stage 2 test --skip tests/ui --skip tests/rustdoc -- --exact
+    SCRIPT: ./x.py check compiletest --set build.compiletest-use-stage0-libtest=true && ./x.py --stage 2 test --skip tests/ui --skip tests/rustdoc -- --exact
```

Comment:
`x check` doesn't actually go through the same codepath as `x build`; I think that for `x check` the stage0 config is not used at all, all the tools are checked with `Mode::ToolRustc` (https://github.com/rust-lang/rust/blob/d3d7a6df5413901041b781480ac61020f36cd552/src/bootstrap/src/core/build_steps/check.rs#L424).

Reply:
Thanks. This specific file contains just tests mock data btw, it doesn't need to be updated :laughing:

Reply:
I am aware, I just wanted them to be 1:1 equal.